main.nf

```diff
@@ -662,6 +673,9 @@ process sample_correlation {
     write.table(hmap\$carpet, 'log2CPM_sample_distances.txt', quote=FALSE, sep="\t")
     
     file.create("corr.done")
+
+    #Printing sessioninfo to standard out
+    sessionInfo()
```

Comment:
Ok, good point. It's once per pipeline, but no easy way to associate it with anything and MultiQC will likely only report one anyway.